keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ResourceRepositoryTests.kt

```diff
@@ -79,14 +80,10 @@ abstract class ResourceRepositoryTests<T : ResourceRepository> : JUnit5Minutests
       Fixture(subject = factory(clock))
     }
 
-//    after { confirmVerified(callback) }
+    after { confirmVerified(callback) }
     after { flush() }
 
     context("no resources exist") {
-      before {
-        flush()
-      }
```

Comment:
This was my workaround, flush the database before the tests because something else wasn't flushing after